test/puppetlabs/puppetdb/command/dlo_test.clj

```diff
@@ -111,8 +113,8 @@
           (is (empty? (archives subdir)))
           (is (empty? (archives other-subdir)))
 
-          (is (= 0 (.count compression)))
-          (is (= 0 (.count failures))))
+          (is (= 0 (timers/number-recorded compression)))
+          (is (= 0.0 (meters/rate-one failures))))
```

Comment:
There is no `.count` method on the object or equivalent retrieval function in the `metrics-clojure` library anymore, so float comparison using the 1 minute rate was the best I could come up with.